dash-renderer/src/store.js

```diff
@@ -19,16 +19,18 @@ const initializeStore = reset => {
 
     const reducer = createReducer();
 
-    // only attach logger to middleware in non-production mode
-    store =
-        process.env.NODE_ENV === 'production' // eslint-disable-line no-process-env
-            ? createStore(reducer, applyMiddleware(thunk))
-            : createStore(
-                  reducer,
-                  window.__REDUX_DEVTOOLS_EXTENSION__ &&
-                      window.__REDUX_DEVTOOLS_EXTENSION__(),
-                  applyMiddleware(thunk)
-              );
+    // eslint-disable-next-line no-process-env
+    if (process.env.NODE_ENV === 'production') {
+        store = createStore(reducer, applyMiddleware(thunk));
+    } else {
+        // only attach logger to middleware in non-production mode
+        const reduxDTEC = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__;
+        if (reduxDTEC) {
+            store = createStore(reducer, reduxDTEC(applyMiddleware(thunk)));
+        } else {
+            store = createStore(reducer, applyMiddleware(thunk));
+        }
+    }
```

Comment:
We've talked about this in DM, can you explain what this is about again?

Reply:
For connecting up the dev bundle to the redux devtools chrome extension, if it happens to be installed. I'm not quite sure when the previous pattern stopped working, the [extension docs](http://extension.remotedev.io/#usage) make it look like it should have been prior to redux 3.1, but maybe the old version kept working until the 4.x bump in #1145 - at any rate prior to my change this would throw an error in dev mode. See also #1166